client/js/tracking.js

```diff
@@ -15,3 +17,29 @@ export default {
     });
   }
 };
+
+function trackOutboundLink(url) {
+  ga('send', 'event', 'outbound', 'click', url, {
+    'transport': 'beacon',
+    'hitCallback': () => {
+      document.location = url;
+    }
+  });
+};
+
+function getDomain(url) {
+  return url.replace('http://', '').replace('https://', '').split('/')[0];
+}
+
+function isExternal(url) {
+  return getDomain(document.location.href) !== getDomain(url);
```

Comment:
Good point. Should we choose what to ignore here (like mailto: and hash links) or do more of an opt in thing – wrapping things that we definitely want to track in e.g. a 'data-track-outboud' block?